postgres_exporter/linux/pgbackrest-info.sh

```diff
@@ -15,7 +15,12 @@ SYSTEM_ID=$1
 [ -f /etc/pgmonitor.conf ] && . /etc/pgmonitor.conf
 
 if [ ${BACKREST_AUTOCONFIG_STANZAS:-0} -gt 0 ]; then
-  BACKREST_STANZAS=$(grep '^\[' /etc/pgbackrest.conf /etc/pgbackrest/  -rh |sort -u |grep -v ':\|global' |sed 's/\[\|\]//g' | tr '\n' ' ' )
+  if [ -f "/etc/pgbackrest.conf" ]; then
+    # Account for old default config file location
+    BACKREST_STANZAS=$(grep '^\[' /etc/pgbackrest.conf /etc/pgbackrest/  -rh |sort -u |grep -v ':\|global' |sed 's/\[\|\]//g' | tr '\n' ' ' )
+  else
+    BACKREST_STANZAS=$(grep '^\[' /etc/pgbackrest/  -rh |sort -u |grep -v ':\|global' |sed 's/\[\|\]//g' | tr '\n' ' ' )
+  fi
```

Comment:
```suggestion
  BACKREST_STANZAS=$(grep '^\[' /etc/pgbackrest.conf /etc/pgbackrest/  -srh |sort -u |grep -v ':\|global' |sed 's/\[\|\]//g' | tr '\n' ' ' )```

Reply:
The `-s` flag suppresses the `file not found` messages from grep, everything else should be the same, yes?

Reply:
oh nice! Yeah that was basically the only issue here.